lms/envs/common.py

```diff
@@ -3216,6 +3216,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # Course Goals
     'lms.djangoapps.course_goals.apps.CourseGoalsConfig',
 
+    # Tagging
```

Comment:
Can we put both `openedx_tagging` and `content_tagging` together, and add an explanatory comment that describes the difference between them? Just from looking at this it's confusing.

Reply:
3ade910d576b8b5b664396f1a85948c284332d15